mocking.md

```diff
@@ -568,7 +568,7 @@ func main() {
 
 If we run the tests and the program manually, we can see that all the behavior remains the same.
 
-Since we are using the `ConfigurableSleeper`,  it is safe to delete the `DefaultSleeper` implementation. Wrapping up our program.
+Since we are using the `ConfigurableSleeper`, it is now safe to delete the `DefaultSleeper` implementation. Wrapping up our program and having a more general Sleeper with arbitrary long countdowns. Adding parameters in such a fashion is called generalization as it makes entities more general and is often followed by encapsulation, where functionality is encapsulated in a function or other entity.
```

Comment:
I'm not sure how helpful this sentence is, it seems to cover a lot of concepts quickly. I'd prefer to remove that.

Reply:
Thanks for reviewing. :) The thing is, the whole chapter covers a large amount of topics, that's why I felt lost a bit and tried to elaborate on the topics covered.

What do you think about referring to Downey's literature the way you did this with Uncle Bobs? I'll make a suggestion later, what I mean.

Reply:
I guess I dont see how encapsulation strongly relates to what we're doing (i can kinda see it, but not sure it adds much). I'll check out your changes when you do them :)

Reply:
Really liked the cake example. It just nails it.